buildtest/buildsystem/base.py

```diff
@@ -498,12 +500,14 @@ def run_tests(self, testfile):
         os.chdir(self._get_testdir())
         self.logger.debug(f"Changing to directory {self._get_testdir()}")
 
-        # Run the test file using the shell
-        cmd = [self.get_shell(), testfile]
+        # command to run the test
+        cmd = []
+        # build the run command that includes the shell path, shell options and path to test file
```

Comment:
If the cmd is created here and then has lines added to it, why do we need to create it as an empty list first? E.g., just do:

```python
cmd = [self.shell.path, self.shell.opts, testfile]
```

Reply:
This is fixed now in https://github.com/shahzebsiddiqui/buildtest-framework/commit/b18e1c53fbb42e081b6e016daa911d63677a9295